plugins/woocommerce/includes/theme-support/class-wc-twenty-twenty-three.php

```diff
@@ -0,0 +1,82 @@
+<?php
+/**
+ * Twenty Twenty-Three support.
+ *
+ * @since   7.0.2
```

Comment:
In many places we specify `@version 7.1.0`, I guess this will need to be unified once we know whether that's included in a patch release?

Reply:
Yup, I'll search and replace it once we figure it out. Good point.

Reply:
Fixed in https://github.com/woocommerce/woocommerce/pull/35306/commits/f543cc05658110ea8d08e3d261d536d0c24d69d5